cernopendata/modules/fixtures/data/records/cms-trigger-examples.json

```diff
@@ -0,0 +1,118 @@
+[
+{
+  "abstract": {
+    "description": " <p>This is an example of the C++ code (and its configuration in Python) needed to extract the trigger information from CMS Open/Legacy data. These instructions are valid to work with 2010 data. Snippets of this software can be used for physics analysis.</p>"
+  }, 
+  "accelerator": "CERN-LHC", 
+  "authors": [
+    {
+      "name": "Carrera, Edgar"
+    }
+  ], 
+  "collections": [
+    "CMS-Tools"
+  ], 
+  "date_created": "2010", 
+  "date_published": "2017", 
+  "distribution": {
+    "formats": [
+      "gz"
+    ],
+    "number_files": 1
+  },
+  "doi": "10.7483/OPENDATA.CMS.TE10.IIC8", 
+  "experiment": "CMS", 
+  "files": [
+    {
+      "checksum": "sha1:3164b6530e1a90fecbe0d742baec61114555b378", 
+      "size": 8759, 
+      "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/trigger_examples-10.0.0.tar.gz"
```

Comment:
`software`, not `luminosity` in the path

Reply:
It moved? Is it this one? `/eos/opendata/cms/software/trigger_examples/trigger_examples-10.0.0.tar.gz`

Reply:
Yes, it's under the `software`, since `luminosity` was wrong.